tests/train_test.py

```diff
@@ -1,23 +1,29 @@
+from unittest import mock
+
 import pytest
+import tensorflow_datasets as tfds
 from click.testing import CliRunner
+from zookeeper.tf.dataset import TFDSDataset
 
 from larq_zoo.training import basic_experiments
-from tests import dummy_datasets  # noqa
 
 
 @pytest.mark.parametrize("command", list(basic_experiments.cli.commands.keys()))
-def test_cli(command):
+@tfds.testing.mock_data(num_examples=2, data_dir="gs://tfds-data/dataset_info")
```

Comment:
Setting `data_dir` to GCS, allows us to read the dataset info without needing to have the metadata files available locally.